projects/frontend/shared-components/gui/projects/shared/src/test.ts

```diff
@@ -25,13 +22,9 @@ declare const require: {
 };
 
 // First, initialize the Angular testing environment.
-getTestBed().initTestEnvironment(
-	BrowserDynamicTestingModule,
-	platformBrowserDynamicTesting(),
-	{
-		teardown: { destroyAfterEach: false }
-	}
-);
+getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting(), {
```

Comment:
A lot of the changes seem to be reformatting of existing files. This is probably due to the linter changes. Not sure this is even a problem, just pointing it out. 🙂

Reply:
I was trying to align code formatting with existing formatting in taurus and supercollider.

Reply:
@DeltaMichael If you agree I could change to spaces and commit whole project in this commit, or we could do it in separate commit and Pull request.

Taurus and Supercollider are working with indent with spaces, 4 for *.ts/*.js and 2 for *.json and also we removed prettier from the projects, because auto formatting is not very good in some cases, and manual formatting is much better and readable. 
To avoid big code style format differences we extracted one configuration file for Intellij that is somewhere in confluence and is a start point that is used by everyone that contribute in UI projects, so we use Intellij auto-format plus manual formatting in some places and that's how we were working in the last years.

Reply:
I'm fine with leaving it as is if that's easier. @ivakoleva Are `*.ts` files included in the GitHub format action?

Reply:
I formatted the whole project `shared-components` with indent space, where 4 spaces for .ts/.js and 2 spaces for *.json, and force prettier to leverage .editorconfig as a source of truth to make sync between our local IDEs and github